### PR TITLE
Amend translations

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/1125189844152671/1188623309021366
Tech Design URL: 
CC: 

**Description**:
This PR adds the new translations from Smartling. There are quite a few changes which include:
- In all the translations the placeholders have been replaced with the type `%1$s`
- Some strings have been amended to remove quotes that were put there by mistake or to remove unnecessary spaces

**Steps to test this PR**:
Check different languages to make sure everything looks good.

Double check the following:
1. Spacing and break lines in settings/about DuckDuckGo look good.
1. Go to `cnn.com` make sure the break lines are good and the dialog shows the emoji correctly.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -352,12 +356,12 @@
     <string name="feedbackIsImportantToUs">Tu opinión anónima nos importa.</string>
 
     <!-- Webview Recovery -->
-    <string name="crashedWebViewErrorMessage">No se pudo mostrar la página web.</string>
-    <string name="crashedWebViewErrorAction">Volver a cargar</string>
+    <string name="crashedWebViewErrorMessage">"No se pudo mostrar la página web."</string>
+    <string name="crashedWebViewErrorAction">"Volver a cargar"</string>
```
Comment (cmonfortep) [inline app/src/main/res/values-es/strings.xml:360]: Noticed these two strings have `"..."`, which is kind of weird. It seems the issue comes from the English strings which were translated with that character.

**Review by cmonfortep — APPROVED** [review]
All looking good, tested in a few languages (English, Spanish, Italian, French): emojis, line breaks, etc. seem right and as expected.

File: app/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (marcosholgado) [reply app/src/main/res/values-es/strings.xml:360]: Yeah, some English strings had `"` and they were missing in the translations. Wether they should have quotes in English or not I have no idea 😅

Comment (cmonfortep) [reply app/src/main/res/values-es/strings.xml:360]: This is the error we show in the snackbar when webview crashes, they don't need quotes. Also `crashedWebViewErrorAction` is the button text. We not related to this PR we can fix it in another one later.

**Pull request merged**